### PR TITLE
some style fixes

### PR DESCRIPTION
**Pull request by mairaw**
While working on the IntelliSense for EventCounter, I stumbled upon this topic. 
If the bold/italic styling is done inside a fenced block, it won't work. 

BTW, is this something we should transform in an official doc? There would be some work involved since this is written in a very informal way.

I also don't know what you mean on item 3 (line 37).

### CODE REVIEW

File: src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md [hunk 1]
```diff
@@ -47,9 +47,7 @@ So, with that, we logged the metric to the `EventCounter`, but unless we can act
 
 There is an extra keyword that you will need to specify the turn on the EventCounters.
 
-```
-PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:**EventCounterIntervalSec=1** collect
-```
+<code>PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:**EventCounterIntervalSec=1** collect</code>
```
Comment (karelz) [inline src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md]: Why code over triple backtick? I thought the triple backtick is preferred in MD files ...

Comment (karelz) [inline src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md]: Ah, I see - the bold in the middle is not bold. It is actually syntax of the command. So we want intentionally to have triple backtick.

**Comment by karelz** [comment]
@mairaw for history: I "only" helped to convert the text into markdown. I apparently left one mistake in, thanks for fixing it!

> I also don't know what you mean on item 3 (line 37).

@vancem wrote the doc, so he can answer.

**Comment by vancem** [comment]
Actually @cshung wrote this guide.   @brianrob  is the official owner.    The formatting change looks fine.  

> I also don't know what you mean on item 3 (line 37).

This is the line you refer to 
```
         //   3. You don't have to have 
```
I agree it is a fragment, my guess is that it should be 
```
         //   3. You don't have to have log with WriteEvent if you don't think you will ever care about details
         //       of individual requests (that counter data is sufficient).  
```
If you want to make that change (or remove the line altogether), that would be in improvement.

**Comment by danmosemsft** [comment]
@mairaw still planning to finish this?

**Comment by karelz** [comment]
@mairaw ping?

File: src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md (hunk 1 above)
Comment (mairaw) [inline src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md]: sorry @karelz, i missed this review earlier. so, you're saying that I should that all \* in that command are part of the syntax? If that's the case, I'll revert to the original.

Comment (karelz) [inline src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md]: Yes, that is correct. Please revert the change.

File: src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md [hunk 2]
```diff
@@ -69,10 +68,8 @@ Notice that, this command also log the events, so we will get both the events an
 
 <img src="PerfViewCapture_Events.png" alt="PerfView Capture of Event Traces" title="PerfView Capture of Event Traces" />
 
-As we mentioned, to avoid overhead, sometimes we will want just the counters. This command can be used to log <em>only</em> the counters:
+As we mentioned, to avoid overhead, sometimes we will want just the counters. This command can be used to log *only* the counters:
 
-```
-PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:*:Critical:EventCounterIntervalSec=1 collect
-```
+<code>PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:*:Critical:EventCounterIntervalSec=1 collect</code>
```
Comment (karelz) [inline src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md]: Same here

**Comment by mairaw** [comment]
Ok, done @karelz!
